dependencies/deployment/github/deployment.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+if [ $# -ne 2 ]; then
+    echo "Usage: bazel run //:deploy-github -- <github-username> <github-token>"
+    exit 1
+fi
+
+set -ex # exit immediately when there's a failure. run the script in verbose mode (ie., print all the commands to stdout)
+
+# command-line arguments
+github_user="$1"
+github_token="$2"
+
+# configurations
+github_repository="cat deployment.properties | grep github.repository | cut -d '=' -f 2"
+distribution_name="grakn-core-all"
+distribution_zipfile_path="dist/grakn-core-all.zip"
+distribution_version=`cat VERSION`
+distribution_basedir_name="$distribution_name-$distribution_version"
+distribution_zipfile_name="$distribution_basedir_name.zip"
+github_tag="v$distribution_version" # e.g, v1.5.0
+ghr_executable_file_name="ghr"
+ghr_basedir_name="ghr_v0.10.2_darwin_386"
```

Comment:
@lolski you are limiting deployment to Darwin-only machines. While this might be fine, this is arbitrary imposed limit. It's better to also support Linux; we have a handy snippet of easily switching between platforms [here](https://github.com/graknlabs/grakn/blob/master/dependencies/tools/unused-deps.sh#L9-L16)

Reply:
nice catch, I haven't fixed this one. Wait for a bit

Reply:
We need to pull in different dependencies depending on whether you're on Mac or on Linux. How would we adjust the code, in your opinion?

```
def dependencies_for_github_deployment():
    native.http_file(
        name = "ghr",
        urls = ["https://github.com/tcnksm/ghr/releases/download/v0.10.2/ghr_v0.10.2_darwin_386.zip"]
    )
```

Reply:
@lolski we should download them as `ghr_mac` and `ghr` and depend (via `sh_binary`'s `data`) on both of them